content/developer/reference/cmdline.rst

```diff
@@ -254,6 +254,11 @@ Emails
 
     Email address used as <FROM> when Odoo needs to send mails
 
+.. option:: --from-filter <address>
+
+    Define for which email address this SMTP configuration can be used (can be empty, a domain name or an entire email address).
+    If the sender do not match this filter, his email will be encapsulated.
```

Comment:
> If the sender's email address does not match this set filter, then the email will not be sent.

It's not True, the email will be sent whatever the form_filter is, it's just used to know if we need to encapsulate or not

Reply:
@std-odoo Okay I am following you, this makes better sense to me now. Thank you!  

Can you explain by what you mean when you say "encapsulate or not".

Reply:
Encapsulate means that the notification email will be used
"Michel" <michel@gmail.com> => "Michel" <notification>@company.com